run-tests.sh

```diff
@@ -24,6 +24,6 @@
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
 pydocstyle invenio_workflows && \
-sphinx-build -qnNW docs docs/_build/html && \
+sphinx-build -qnN docs docs/_build/html && \
 python setup.py test && \
-sphinx-build -qnNW -b doctest docs docs/_build/doctest
+sphinx-build -qnN -b doctest docs docs/_build/doctest
```

Comment:
For the tox.ini, you have to put each requirement on a separate line (for what I could figure out):
```
    {docs,docstyle,doctests}: Sphinx>=1.5.1
    {docs,docstyle,doctests}: Celery<4.0
```